apps/src/gamelab/AnimationPicker/index.jsx

```diff
@@ -0,0 +1,17 @@
+var AnimationPickerBody = require('./AnimationPickerBody.jsx');
+var Dialog = require('../../templates/Dialog.jsx');
+
+var AnimationPicker = React.createClass({
+  propTypes: {
+    handleClose: React.PropTypes.func.isRequired
+  },
+
+  render: function () {
+    return (
+      <Dialog handleClose={this.props.handleClose} isOpen>
+        <AnimationPickerBody />
+      </Dialog>
+    );
+  }
+});
```

Comment:
Is this going to get more complex? If not, it could be a functional stateless component